pkg/catalogv2/helm/helm2.go

```diff
@@ -135,6 +147,7 @@ func toMap(namespace, name string, manifest string) map[string]interface{} {
 	return values
 }
 
+// decodeHelm2 receives a helm2 release data and returns the corresponding k8s release proto struct
```

Comment:
// decodeHelm2 receives a helm2 release data and returns the corresponding helm2 release proto struct